engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -221,6 +233,19 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
       log.info(s"$tag transitioning from $fromState to $toState.")
   }
 
+  private def handleRetryableFailure(jobKey: BackendJobDescriptorKey) = {
```

Comment:
I believe currently (and the in the future) we want to treat these as separate calls and not overwrite them, at the very least from a metadata perspective, so that after the fact you know things about the failed tasks (how long they ran, etc)